homeassistant/components/xiaomi_ble/config_flow.py

```diff
@@ -189,7 +189,7 @@ async def async_step_confirm_slow(
         self, user_input: dict[str, Any] | None = None
     ) -> FlowResult:
         """Ack that device is slow."""
-        if user_input is not None or not onboarding.async_is_onboarded(self.hass):
+        if user_input is not None:
```

Comment:
Unintentional change?

Reply:
It was a c+p error when i copied `async_step_confirm_slow` from  `async_step_bluetooth_confirm`. I don't think that check applies to that flow path, but I did forget to include it in the description.

Reply:
(Updated description)